c7n/resources/lakeformation.py

```diff
@@ -11,5 +12,6 @@ class resource_type(TypeInfo):
         arn = id = 'ResourceArn'
```

Comment:
this line also needs to change as it's referencing the resource arn not the lakeformation arn

Reply:
Hmm, afaict the lakeformation resource arn isn't its own arn, it references a related s3 resource. So should we add `arn_type = ''` and lean into that (which does seem to work), or remove the tagging functionality altogether and explicitly say that `lakeformation-resource` has no arn or direct tagging support?

Reply:
it looks like the tag operation is on the bucket itself, which in that case means we're just doing a s3 tagging operation which the s3 resource already supports. it's not clear to me how this is different from that. If the goal is to tag buckets that are lakeformation managed can't you just use the tag:LakeFormationManaged to filter for those buckets with the s3 resource?

Reply:
Yeah, you definitely could. I am perhaps reading too much into the original PR's inclusion of tagging support. Thinking they'd want to say "find all lakeformation resources and tag the underlying buckets". Maybe we should just keep that on the s3 resource though? I'll remove the support from here.

Reply:
Ok made those changes.

I wonder if stripping tag support out of this resource implies that we should add a lakeformation related resource filter to s3. I guess that's a separate issue though, where the immediate concern is not releasing a resource that will be broken and force backward-incompatible changes.

Reply:
arn_type = '' seems good. afaics the lake formationg tagging applies to the resource in the lake formation api, not the underlying bucket?

Reply:
well if it's using the resource groups tagging api and the resource arn is the s3 bucket the bucket itself would be tagged right?

Reply:
Yeah, the universal tagging filter/action would hit the resource arn which (at least today) refers to s3. One wrinkle actually, today that resource could be an s3 bucket or a path within the bucket. So I guess if we go back to `arn_type = ''` and let lakeformation-resource tag those underlying s3 buckets, we'd also need a `get_arns` implementation that trims off the path bits.

Reply:
so this shouldn't be tangible. ie. one resource managing tags on a different resource seems a recipe for confusion. The lake formation tag api highlights half dozen service owned domain objects, https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lakeformation.html#LakeFormation.Client.add_lf_tags_to_resource

 I think part of the issue here is its not clear that this is a valid lake formation resource re representing the s3 bucket. it seems like it would be better served as an s3 bucket filter.